config/kafka/server.properties

```diff
@@ -0,0 +1,48 @@
+#Listener config
+listeners=auth-listener-docker-internal://kafka:19092,auth-listener-docker-external://0.0.0.0:9092,listener-docker-internal://kafka:19093,listener-docker-external://0.0.0.0:9093,redpanda://kafka:19094
```

Comment:
Maybe document WHAT we are doing.

e.g. setting up 3? listernes, one for blah...

Reply:
Could also be in readme.md

Reply:
put it in readme if the text is more than one paragraph